cibyl/models/ci/system_factory.py

```diff
@@ -57,4 +58,7 @@ def create_system(system_type, name, **kwargs):
         if system_type == SystemType.ZUUL:
             return ZuulSystem(name=name, system_type=system_type, **args)
 
+        if system_type == SystemType.ZUULD:
+            return JobsSystem(name=name, system_type=system_type, **args)
```

Comment:
Once we have the '--tenants' option, you will probably want to change this for a ZuulSystem. JobsSystem is meant for Jenkins and ignores the tenants level that Zuul has.

Reply:
Done